### ISSUE TITLE
pulp --watch browserify error

### ISSUE THREAD

**Issue by mjhoy** [purescript-contrib/pulp#300]
I've recently updated to pulp 12.0.0 and get this error:

    ERROR: Internal error in Pulp.Args.Get: Missing default value for option: _project

when running `--watch browserify`.

Here's my shell session:

```sh
[mjhoy@mjh-mbp client]$ PATH=$PATH:./node_modules/.bin pulp --version
Pulp version 12.0.0
purs version 0.11.6 using node_modules/.bin/purs

[mjhoy@mjh-mbp client]$ PATH=$PATH:./node_modules/.bin pulp --watch browserify --to app.js
* ERROR: Internal error in Pulp.Args.Get: Missing default value for option: _project
This is a bug. Please report it.

[mjhoy@mjh-mbp client]$ * Browserifying project in /Users/mjhoy/proj/frootloop/client
* Building project in /Users/mjhoy/proj/frootloop/client
* Build successful.
* Browserifying...
* Browserified.

[mjhoy@mjh-mbp client]$
```

(the second command seems to spawn a process that dies, but it still browserifies successfully.)

**Comment by mjhoy** [comment]
I have also switch to `psc-package` from bower, perhaps this is related.

**Comment by hdgarrood** [comment]
Does it work if you switch to bower?

**Comment by mjhoy** [comment]
If I bring back my `bower.json`, I get the same error.

**Comment by mjhoy** [comment]
I should add that without the watch, browserify works fine.

**Comment by hdgarrood** [comment]
I've published @thoradam's fix as v12.0.1, you can run `npm install [-g] pulp@next` to get it. Thanks for the report!
